apps/prairielearn/src/sync/course-db.js

```diff
@@ -758,6 +762,33 @@ export async function loadCourseInfo(coursePath) {
   const topics = getFieldWithoutDuplicates('topics', 'name');
   const assessmentModules = getFieldWithoutDuplicates('assessmentModules', 'name');
 
+  // In production environments, check if any features specified in `devModeFeatures`
+  // aren't also enabled for the course. If so,
```

Comment:
Comment ends unexpectedly!

Reply:
Good catch! I removed it entirely in a818818, I think it's redundant given comments on the following lines.